NEWS.md

```diff
@@ -1,7 +1,9 @@
 ## 4.4.1 (IN PROGRESS)
 
  * Accounts without item data attached no longer result in errors when using "includeCharges" flag (MODPATRON-59)
- 
+ * Accounts now include correct accrual date when using the "includeCharges" flag (MODPATRON-62)
+ * Requires `circulation 9.5 10.0 or 11.0` ([MODPATRON-52](https://issues.folio.org/browse/MODPATRON-52), [MODPATRON-54](https://issues.folio.org/browse/MODPATRON-54))
```

Comment:
I don't think this change is included in this fix that is being back ported.

Reply:
I just removed it.  Not sure where it came from.

Reply:
It comes from the way that git tracks snapshots of files, so any changes to a file that was changed in that commit get included.